file name error

```
\user.java:1: error: class User is public, should be declared in a file named User.java
public class User {
             ^
1 error
```


please fix the error

--- a/user.java
+++ b/user.java
@@ -1,15 +1,14 @@
-public class User {
+public class user {
     private int id;
     private String username;
     private String role;
 
-    public User(int id, String username, String role) {
+    public user(int id, String username, String role) {
         this.id = id;
         this.username = username;
         this.role = role;
     }
 
-    // Getters
     public int getId() { return id; }
     public String getUsername() { return username; }
     public String getRole() { return role; }

